### PR TITLE
UweDrechsel.vym version 2.8.8

### PR DESCRIPTION
**Pull request by chausner**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18686)

### CODE REVIEW

**Comment by ChuckFerring** [comment]
It looks like the issue is that the installer is specified as x86 but may actually be x64. Did you specify the correct architecture?

**Comment by chausner** [comment]
> It looks like the issue is that the installer is specified as x86 but may actually be x64. Did you specify the correct architecture?

I think you are right. The installer installs into Program Files (x86) but the application is actually x64. I changed in the architecture in the manifest.

File: manifests/u/UweDrechsel/vym/2.8.8/UweDrechsel.vym.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: UweDrechsel.vym
+PackageVersion: 2.8.8
+PackageLocale: en-US
+Publisher: Uwe Drechsel
+PackageName: vym
```
Comment (OfficialEsco) [inline manifests/u/UweDrechsel/vym/2.8.8/UweDrechsel.vym.locale.en-US.yaml:8]:
```suggestion
PackageName: vym - View Your Mind
```

**Review by OfficialEsco — APPROVED** [review]
Might have to change this later, but it should work anyways

**Pull request merged**